modules/local/tabix_sort_bgzip.nf

```diff
@@ -0,0 +1,41 @@
+// Modified version of my TABIX_BGZIP that sorts the input file first.
+// Supports BED and GFF files.
+process TABIX_SORT_BGZIP {
```

Comment:
I was not planning too :pleading_face: but I should. However, looking into it, I would use https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1930-3 rather than a regular sort.